tests/python_package_test/test_dask.py

```diff
@@ -96,6 +234,8 @@ def test_classifier(output, centers, client, listen_port):
     assert_eq(y, p2)
     assert_eq(p1_proba, p2_proba, atol=0.3)
 
+    client.close()
```

Comment:
@jameslamb Are you agree with this unrelated to Ranker itself change?

Reply:
Yes, I'm ok with this. I'm not 100% sure that it is unrelated, even though it's touching non-ranker tests. Since this PR is close to being merged and since there aren't other PRs waiting on this fix (the timeout / shutdown issues in Dask tests have not been as severe elsewhere), I'm ok leaving it.